public/css/footer.css

```diff
@@ -23,6 +23,10 @@
 	color: var(--green);
 }
 
+.myFooter i.fa {
```

Comment:
Minor nitpick, but you can get rid of the `i` in `i.fa` here, so `.myFooter .fa`. I find that clearer personally since the former makes me go, "wait is there an element that has class `fa` but is not an `i` in the footer?"